makepanda/makewheel.py

```diff
@@ -0,0 +1,195 @@
+Root-Is-Purelib: false
+Tag: py2-none-any
+Tag: py3-none-any
+Build: 1
+"""
+
+METADATA = {
+    "license": "BSD",
+    "name": "Panda3D",
+    "metadata_version": "2.0",
+    "generator": "makepanda",
+    "summary": "Panda3D is a game engine, a framework for 3D rendering and "
+               "game development for Python and C++ programs.",
+    "extensions": {
+        "python.details": {
+            "project_urls": {
+                "Home": "http://www.panda3d.org/"
+            },
+            "document_names": {
+                "license": "LICENSE.txt"
+            },
+            "contacts": [
+                {
+                    "role": "author",
+                    "email": "etc-panda3d@lists.andrew.cmu.edu",
+                    "name": "Panda3D Team"
+                }
+            ]
+        }
+    },
+    "classifiers": [
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Intended Audience :: End Users/Desktop",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: OS Independent",
+        "Programming Language :: C++",
+        "Programming Language :: Python",
+        "Topic :: Games/Entertainment",
+        "Topic :: Multimedia",
+        "Topic :: Multimedia :: Graphics",
+        "Topic :: Multimedia :: Graphics :: 3D Rendering"
+    ]
+}
+
+
+def write_metadata(info_target):
+    details = METADATA["extensions"]["python.details"]
+    homepage = details["project_urls"]["Home"]
+    author = details["contacts"][0]["name"]
+    email = details["contacts"][0]["email"]
+    with open(join(info_target, 'METADATA'), 'w') as outfile:
+        outfile.writelines([
+            "Metadata-Version: {metadata_version}",
+            "Name: {name}",
+            "Version: {version}",
+            "Summary: {summary}",
+            "License: {license}".format(**METADATA),
+            "Home-page: {}".format(homepage),
+            "Author: {}".format(author),
+            "Author-email: {}".format(email),
+            "Platform: {}".format(PLATFORM_VERSION),
+        ] + ["Classifier: {}".format(c) for c in METADATA['classifiers']])
+
+
+def zip_directory(path, zip_handler):
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            full_path = os.path.join(root, file)
+            short_path = full_path.replace(path, '')
+            zip_handler.write(full_path, short_path)
+
+
+def copy_files(src, target):
+    """Copy all files from a directory to another. Differs from copytree
+    because it copies the files contained, not the directory itself.
+    """
+    for f in os.listdir(src):
+        shutil.copy2(join(src, f), join(target, f))
+
+
+def makewheel(version):
+    # Global parameters that depend on version
+    data_target = join(WHEEL_TARGET, "panda3d-{}.data".format(version))
+    info_target = join(WHEEL_TARGET, "panda3d-{}.dist-info".format(version))
+    wheel_name = "panda3d-{}-{}-none-{}.whl".format(
```

Comment:
Note here that the wheel wouldn't install unless the ABI tag was "none". However, I did adjust the python version to explicitly require cpython.